packages/qwik/src/core/component/component.public.ts

```diff
@@ -186,25 +181,7 @@ export const componentQrl = <PROPS extends Record<any, any>>(
   componentQrl: QRL<OnRenderFn<PROPS>>
 ): Component<PROPS> => {
   // Return a QComponent Factory function.
-  // This is only used in v1, in v2 we use componentQrl directly
-  function QwikComponent(props: PublicProps<PROPS>, key: string | null, flags: number): JSXNode {
-    assertQrl(componentQrl);
-    assertNumber(flags, 'The Qwik Component was not invoked correctly');
-    const hash = qTest ? 'sX' : componentQrl.$hash$.slice(0, 4);
-    const finalKey = hash + ':' + (key ? key : '');
-    return _jsxSorted(
-      Virtual,
-      {
-        props,
-        [OnRenderProp]: componentQrl,
-        [QSlot]: props[QSlot],
-      },
-      null,
-      props.children,
-      flags,
-      finalKey
-    ) as any;
-  }
```

Comment:
for some reason for the qwik-react here is returning error. This is not needed in v2, so I removed it